src/expr.c

```diff
@@ -4150,11 +4150,15 @@ static int exprCodeVector(Parse *pParse, Expr *p, int *piFreeable){
   }else{
     *piFreeable = 0;
     if( p->op==TK_SELECT ){
+#ifdef _FREEBSD_KERNEL
+      iResult = 0;
+#else
 #if SQLITE_OMIT_SUBQUERY
```

Comment:
Why not define SQLITE_OMIT_SUBQUERY as part of a general #ifdef _FREEBSD_KERNEL in the root include file?  Whichever .h file is use to configure much of SQLite?

Reply:
I had initially done this, but several other errors with functions the kernel could not support emerged from other files. This is why SQLITE_OMIT_SUBQUERY was silenced and the code was modified so that I was not forced to define it.